akalisten/templates/lotsude/style.css

```diff
@@ -217,10 +223,42 @@ footer {
 .maybe h2 { color: #f39c12; border-color: #f39c12; }
 .highlight { border-color: #f39c12; }
 
+.lucide-icon {
+    color: var(--icon);
+    width: 1em;
+    height: 1em;
+}
+.lucide-icon-15 {
+    color: var(--icon);
+    width: 1.5em;
+    height: 1.5em;
+}
+.lucide-icon-2 {
+    color: var(--icon);
+    width: 2em;
+    height: 2em;
+}
+.inline-icon {
+    margin-right: 1ex;
```

Comment:
Instead of using manual margins, I would use [`gap`](https://developer.mozilla.org/en-US/docs/Web/CSS/gap) as a general definition in `.icon-text` or similar.

Reply:
Ah, that works well. Thanks for the tip!